clients/src/main/resources/common/message/ReadShareGroupStateResponse.json

```diff
@@ -42,7 +42,8 @@
           "about": "The state epoch for this share-partition." },
         { "name": "StartOffset", "type": "int64", "versions": "0+",
           "about": "The share-partition start offset, which can be -1 if it is not yet initialized." },
-        { "name": "StateBatches", "type": "[]StateBatch", "versions": "0+", "fields":[
+        { "name": "StateBatches", "type": "[]StateBatch", "versions": "0+",
+          "about": "The state batches for this partition.", "fields":[
```

Comment:
nit: for consistency should we say `share-partition` instead of just `partition`?